code/modules/admin/verbs/ping_all_admins.dm

```diff
@@ -0,0 +1,38 @@
+/client/proc/ping_all_admins()
+	set name = "Ping all admins"
+	set category = "Admin"
+
+	if(!check_rights(R_ADMIN, FALSE))
+		return
+
+	var/msg = input(src, "What message do you want the ping to show?", "Ping all admins") as text|null
+	msg = sanitize(copytext(msg, 1, MAX_MESSAGE_LEN))
+
+	if(!msg)
+		return
+
+	var/list/admins_to_ping = list()
+
+	for(var/client/C in GLOB.admins)
+		if(R_ADMIN & C.holder.rights)
+			admins_to_ping.Add(C)
+
+	var/de_admin_also = alert(usr, "Do you want to ping admins that have used de-admin?","Ping all admins", "Yes", "No")
+	if(de_admin_also == "Yes")
+		for(var/key in GLOB.de_admins)
+			var/client/C = get_client_by_ckey(key)
+			if(!istype(C))
+				continue
+			admins_to_ping.Add(C)
+
+	if(length(admins_to_ping) < 2) // All by yourself?
+		to_chat(usr, "<span class='boldannounce'>No other admins online to ping[de_admin_also == "Yes" ? ", including those that have used de-admin" : ""]!</span>")
+		return
+
+	var/datum/asays/asay = new(usr.ckey, usr.client.holder.rank, msg, world.timeofday)
+	GLOB.asays += asay
+	log_ping_all_admins(msg, src)
+
+	for(var/client/C in admins_to_ping)
+		SEND_SOUND(C, sound('sound/misc/ping.ogg'))
+		to_chat(C, "<span class='admin_channel'>ALL ADMIN PING: <span class='name'>[key_name(usr, 1)]</span> ([admin_jump_link(mob)]): <span class='message'>Your attention is required in asay. <font color='red'>[length(admins_to_ping)] clients pinged.</font> Reason: <span class='emoji_enabled'>[msg]</span></span></span>")
```

Comment:
I would just make this <big>, and drop the "your attention is required in asay" part. Make it just show a normal message but bigger and maybe a different more obvious colour.